### PR TITLE
Rs 2022.12.10.1

### PR DESCRIPTION
**Pull request by rschlaikjer**
The save juggling begins
2022-12-10-173130_1920x1140_scrot

### CODE REVIEW

File: script/Ciel/Day 12/12_01_CIEL12_2B.txt [hunk 1]
```diff
@@ -312,12 +331,13 @@ Ten minutes left until the next class.
...
 -- 昼休みを待っていられない。自分でも浮かれすぎていると分かっているが止められない。
-I can't wait until lunch break. I know I'm being way too overeager, but I can't hold back.
+There's no way I can hold out until lunch.
+ I know I'm being way too overeager, but I can't help myself.
 }
 [sha:789e49efbf33584b1a285126b03cde28cf95600c]{
 -- Page 51, Offset 30994.
 -- 顔を見るだけでいいから、三年の教室にいる先輩を探しに行こう……！
-Just catching a glimpse of her will be enough. I'll head to the third-year classroom and find Senpai...!
+Just catching a glimpse of her will be enough. I'll head to the third-year classrooms and find Senpai...!
```
Comment (Petrikow) [inline script/Ciel/Day 12/12_01_CIEL12_2B.txt:340]: ellipsis

File: script/Ciel/Day 12/12_01_CIEL12_2B.txt [hunk 2]
```diff
@@ -367,12 +389,14 @@ You're always the one doing the visiting, so I figured I'd head over to you for
...
-It feels like you're pointing out something shameful."
+I feel like I'm quietly being judged for something."
 }
 [sha:090e5a5170d1a2bb722219aef2c65a51f49cdc3e]{
 -- Page 56, Offset 31005.
 -- 「いや、茶道部の部費でお茶飲んでるだけじゃなくて良かったなと。うん、感心感心。分かってましたけど、真面目に授業を受けてたんですね」
-"No, I'm glad you're not just wasting the tea ceremony club's budget and drinking tea all the time. Yes, how very admirable. I already knew this beforehand, but you really are taking school seriously."
+"No, I'm simply glad to hear you don't just sit around and drink tea on the tea ceremony club's dime.
+ Yeah, I've got to hand it to you.
+ You're really taking school seriously. Not that I had any doubts, of course."
```
Comment (Petrikow) [inline script/Ciel/Day 12/12_01_CIEL12_2B.txt:399]: You're really > You really are

She said at some point earlier that she actually does take school seriously when they're talking (a lie, but nevertheless), so this is him affirming her statement.

File: script/Ciel/Day 12/12_01_CIEL12_2B.txt [hunk 3]
```diff
@@ -367,12 +389,14 @@ You're always the one doing the visiting, so I figured I'd head over to you for
...
 }
 [sha:263b6c6b8f924283b070198a09a256afb868b0bc]{
 -- Page 57, Offset 31006. Mods: @k, @e.
```
Comment (Petrikow) [inline script/Ciel/Day 12/12_01_CIEL12_2B.txt:402]: 31009 should probably be like "Oops" or something to that effect.

File: script/Ciel/Day 12/12_01_CIEL12_2B.txt [hunk 4]
```diff
@@ -397,27 +421,34 @@ Tohno-kun, is that really how you saw me?!"
 [sha:5686a1204a3071a65046da59fe37c00602a4b8d0]{
 -- Page 57, Offset 31010.
 -- ……しまった。微塵も思っていなかった、と断言できないのが我ながら薄情だ。
-...I messed up. The fact that I can't completely deny it is heartless, even for me.
+I messed up...
+ The fact that I can't bring myself to pretend I didn't mean it is a little heartless, even for me.
```
Comment (Petrikow) [inline script/Ciel/Day 12/12_01_CIEL12_2B.txt:425]: "I messed up..." could use a replacement. It feels too descriptive, and not evocative of his feelings.

>Shouldn't have said that.

>Damnit...

File: script/Ciel/Day 12/12_01_CIEL12_2B.txt [hunk 5]
```diff
@@ -397,27 +421,34 @@ Tohno-kun, is that really how you saw me?!"
...
 }
 [sha:0b8d4078d097a36e6a07c4e3a1b2a7dac2ea2658]{
 -- Page 58, Offset 31011.
 -- 「……すみません、あまりに先輩が神出鬼没なんで、良からぬ想像をする時もなくはありませんでした。先輩の耳には届いてないだろうけど、一、二年の間じゃ楽しい噂話もあるし」
-"...I'm sorry, but with how elusive you are, it's pretty hard to not make bad assumptions about you. I doubt you've heard about it, but some interesting rumors are making the rounds among the first and second-year students."
+"I'm sorry, but given how liable you are to pop up unexpectedly,
+ people have started to come up with some pretty crazy theories about you.
+ They probably haven't reached your ears yet,
+ but there are some interesting rumors making the rounds among the first and second-year students."
```
Comment (Petrikow) [inline script/Ciel/Day 12/12_01_CIEL12_2B.txt:433]: 神出鬼没 isn't just about popping up, but also the places
>but given how liable you are to show up where you'd least except...

Then there's the second clause, which when edited like this clashes with the second sentence.
Logic: "Ciel is elusive > hard not to let the imagination run wild. Therefore, these students are even spreading some interesting rumours"

>you can't fault people for coming up with some pretty odd theories about you.

just add a little 'actually' for the もあるし mood.
>but there are _actually_ some pretty interesting rumors making the rounds among the first and second-year students

File: script/Ciel/Day 12/12_01_CIEL12_2B.txt [hunk 6]
```diff
@@ -397,27 +421,34 @@ Tohno-kun, is that really how you saw me?!"
...
 }
 [sha:41aa2d2978e96c60e684823f302d60eaaa52bcfe]{
 -- Page 58, Offset 31012.
 -- 「ほほう。良からぬ想像とは、どのような？」
-"Really now? And what exactly are these bad assumptions?"
+"Oh really? Can you give my an example of said theories?"
```
Comment (Petrikow) [inline script/Ciel/Day 12/12_01_CIEL12_2B.txt:438]: Can you > Could you

File: script/Ciel/Day 12/12_01_CIEL12_2B.txt [hunk 7]
```diff
@@ -11,22 +11,27 @@ With school drawing near, the road is teeming with students
...
+I can spot a couple people that seem be struggling with the return of the workweek,
+ but the overall mood is still cheerful.
+ The bizarre events that had plagued the city seem to have stopped,
+ leaving everyone feeling like a weight has been lifted from their shoulders.
 }
 [sha:8711c4151ad7cee07513d976eb74aa18a026eb3e]{
 -- Page 28, Offset 30934.
 -- そんな周りの様子に俺はまだ溶け込めない。
-But I've yet to adjust to the surrounding mood.
+And yet, I can't adjust to this carefree atmosphere.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 12/12_01_CIEL12_2B.txt:22]: Sounds right without it imo

**Pull request merged**